src/AppConstants.js

```diff
@@ -59,3 +59,13 @@ export const FILTER_CATEGORIES = [
         ]
     }
 ];
+export const RISK_OF_CHANGE_DESC = {
+    1: 'The change takes very little time to implement and there is minimal impact to system operations. These are typically configuration changes that do not impact a running kernel or service. These changes can be made by modifying a line or file; the change that is taken is immediate and does not impact the system itself.', // eslint-disable-line max-len
```

Comment:
`// eslint-disable-line max-len` should be done once, at the top of the file

Reply:
:+1: